apps/coscrad-frontend/src/components/resource-info/presenters/resource-info.presenter.tsx

```diff
@@ -15,16 +16,25 @@ IAggregateInfo<ResourceType> & { route: string }): JSX.Element => (
     <Link to={`/${route}`}>
         <Card>
             {/* TODO Handle pluralization properly as soon as we have a Resource Type whose plural form is irregular */}
-            <CardHeader title={label} />
+            {/* <CardHeader title={label} /> */}
```

Comment:
Are we putting the title back in later? If not, just remove the commented out code.

Reply:
Removed